docs/api/structures/offscreen-shared-texture.md

```diff
@@ -4,21 +4,15 @@
   * `widgetType` string - The widget type of the texture. Can be `popup` or `frame`.
   * `pixelFormat` string - The pixel format of the texture. Can be `rgba` or `bgra`.
   * `codedSize` [Size](size.md) - The full dimensions of the video frame.
-  * `visibleRect` [Rectangle](rectangle.md) - A subsection of [0, 0, codedSize.width(), codedSize.height()]. In OSR case, it is expected to have the full section area.
+  * `colorSpace` [ColorSpace](color-space.md) - The color space of the video frame.
```

Comment:
We expose `colorSpace` as a string in the Display object which is a bit unfortunate. I think it's fine to expose with this struct here though. 👍 
https://www.electronjs.org/docs/latest/api/structures/display

Later we can figure out whether we want to expose the struct within Display under a potentially different property name.